examples/overwriteFamily.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script demonstrates how to run the and use the overwriteFamilies option
```

Comment:
I would just say "family option" because the command line argument is called `family`.

Reply:
Yeah, I thought about this. In some sense, I think the `family` options should really be `overwriteFamily`, since I think that reflects the option more explicitly. We can address that later. I renamed the script too to reflect the option.

Reply:
I agree that it would be best to rename the options at some point.